src/api/v1beta1/activegate_types.go

```diff
@@ -42,13 +42,20 @@ var (
 		ShortName:    "dynatrace-api",
 		ArgumentName: "restInterface",
 	}
+
+	StatsDIngestCapability = ActiveGateCapability{
+		DisplayName:  "statsd-ingest",
+		ShortName:    "statsd-ingest",
+		ArgumentName: "extension_controller",
+	}
 )
 
 var ActiveGateDisplayNames = map[CapabilityDisplayName]struct{}{
 	RoutingCapability.DisplayName:       {},
 	KubeMonCapability.DisplayName:       {},
 	MetricsIngestCapability.DisplayName: {},
 	DynatraceApiCapability.DisplayName:  {},
+	//StatsDIngestCapability.DisplayName: {}, // TODO Enable once EEC/StatsD images are pullable from the cluster
```

Comment:
this should be uncommented so it can be enabled

Reply:
Actually, it's commented out in a separate commit on purpose. I'd rather enable it no sooner than we have a confirmation EEC/StatsD images are integrated with Dynatrace Docker registry which won't happen until sprint 237+.

Reply:
This is enabled already via feature flag, hiding the capability makes no difference

Reply:
It actually _does_ make a difference because feature flags are only discoverable through code inspection whereas CR properties are discoverable with `kubectl explain dynakube.spec.activeGate`.

Either way, to keep things simple I'll just remove the flag and enable capability by default to avoid confusion and will worry when and if it's needed again.